cpp/src/arrow/util/logging.cc

```diff
@@ -116,7 +116,7 @@ static std::unique_ptr<std::string> log_dir_;
 #ifdef ARROW_USE_GLOG
 
 // Glog's severity map.
-static int GetMappedSeverity(ArrowLogLevel severity) {
+static google::LogSeverity GetMappedSeverity(ArrowLogLevel severity) {
```

Comment:
Does this break compatibility with previous versions? If so, perhaps use `auto` to be more lenient.

Reply:
Ok, I see that previously it was `using LogSeverity = int`, so nothing should break. Thank you.